force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -162,10 +170,8 @@ public with sharing class ClassesObjectsMethods {
      * @return A integer between 50 and 100.
      */
     public static Integer generateRandomNumber() {
-        // Use Math class to generate number between 50 and 100
-
-        // Return the random integer
-        return null; // Replace null with the variable you used to store the result
+        Integer result = Integer.valueOf((Math.random() * 50) + 50);
```

Comment:
This formula looks good except that it will only give you a range between 50 and 99. To get to 100, you need to add 1 to the inner number:
```suggestion
        Integer result = Integer.valueOf((Math.random() * 51) + 50);
```
**Integer result = Integer.valueOf((Math.random() * max - min + 1) + min);**